browser/ui/tabs/brave_tab_color_mixer.cc

```diff
@@ -61,13 +63,17 @@ void AddBraveTabPrivateThemeColorMixer(ui::ColorProvider* provider,
                                        const ui::ColorProviderKey& key) {
   auto& mixer = provider->AddMixer();
   mixer[kColorBraveVerticalTabActiveBackground] = {
-      mixer.GetResultColor(kColorTabBackgroundActiveFrameActive)};
+      nala::kColorPrimitivePrivateWindow30};
   mixer[kColorBraveVerticalTabInactiveBackground] = {
       mixer.GetResultColor(kColorToolbar)};
+  mixer[kColorBraveVerticalTabHoveredBackground] = {
+      ui::AlphaBlend(kColorBraveVerticalTabActiveBackground,
+                     kColorBraveVerticalTabInactiveBackground,
+                     /* 40% opacity */ 0.4 * SK_AlphaOPAQUE)};
```

Comment:
@aguscruiz  Re-used previously used color for hover color but maybe we want to use nala token for this?

Reply:
If it doesn't break anything, then sure, I'm all for that.